test/test_dds.py

```diff
@@ -11,10 +11,10 @@
 import unittest
 import threading
 
-from .utilities import nesw_to_dds_format
-from .utilities import rotate_nesw_to_eswn
-from .utilities import run_in_threads
-from .utilities import check_DD_table_results
+from test.utilities import nesw_to_dds_format
+from test.utilities import rotate_nesw_to_eswn
+from test.utilities import run_in_threads
+from test.utilities import check_DD_table_results
```

Comment:
I was wondering if it should `from . import function` but then I noticed it would require `__init__,py` to import everything. Then I landed to [https://stackoverflow.com/questions/13392038/](https://stackoverflow.com/questions/13392038/) which indicates there should be only one method to import python package.

So in end it looks like this method is the correct way to import locally.